### ISSUE TITLE
Could I use PostgreSQL now?

### ISSUE THREAD

**Issue by cmdntd** [RunOnFlux/Flux-Shared-DB#54]
Could I use PostgreSQL now?
Do you have any roadmap for this?

**Comment by alihm** [comment]
No it's not, We're working on a new shared-db version for Arcane that will utilize built-in db replication. New version will support most DBs that have built-in replication.

**Comment by batonac** [comment]
That's great news! I wonder if [YugabyteDB](https://github.com/yugabyte/yugabyte-db) or [pgEdge](https://github.com/pgEdge/pgedge) could be adapted to run on Flux.

**Comment by alihm** [comment]
Absolutely! After Arcane release all cloud native DBs can be adopted on Flux.

**Comment by cmdntd** [comment]
How does encryption prevent node provider from interfering with this database on flux?

**Comment by alihm** [comment]
There will be a sandboxed OS where node owners have limited access just to maintain the node. And all data will be encrypted at-rest.
